### ISSUE TITLE
[REQUEST] Add limiters for Standart Tool

### ISSUE THREAD

**Issue by PROPHESSOR** [markusschwenk/midieditor#39]
For example, so that holding the Ctrl key allows events to be moved only vertically, and Shift - only horizontally.
This will allow you to completely get rid of unnecessary tools without losing their capabilities. (#30)

**Comment by robaxeman** [comment]
Good idea. This would make editing quicker without having to swap tool all the time.
